crates/sui-graphql-rpc/Cargo.toml

```diff
@@ -14,11 +14,15 @@ async-trait.workspace = true
 axum.workspace = true
 chrono.workspace = true
 clap.workspace = true
+env_logger = { version = "0.10.0" }
 fastcrypto = { workspace = true, features = ["copy_key"] }
+futures-util = {version = "0.3.17"}
```

Comment:
Not really experienced with tokio and futures, but couldn't tokio Mutex be used here?

Reply:
Yeah I think so